actix-web-codegen/src/lib.rs

```diff
@@ -141,6 +141,21 @@ pub fn patch(args: TokenStream, input: TokenStream) -> TokenStream {
     route::generate(args, input, route::GuardType::Patch)
 }
 
+/// Creates route handler with Multiple HTTP methods guards.
+///
+/// Syntax: `#[route("path"[, attributes])]`
```

Comment:
Part of this doc block should clearly show that this macro is designed to be used with multiple methods.

Reply:
I've added an `Example` line, similar to the `Syntax` line.